### PR TITLE
bad php

### PR DESCRIPTION
**Pull request by gisforgirard**
someone could probably write it better but i tested it and it doesnt crash or show warnings like before so its the best you're getting for free right now lol

- [ ] Redmine Issue: https://redmine.pfsense.org/issues/NNNN
- [ ] Ready for review

### CODE REVIEW

**Comment by jim-p** [comment]
Nothing really usable here. Please open an issue at https://redmine.pfsense.org/ with the full text of the PHP errors you saw originally.

**Comment by gisforgirard** [comment]
honestly i dont get paid enough to care sorry

**Comment by gisforgirard** [comment]
Dope. I was just trying to let it be known that code needed a fix (it was
fatal erroring a blank white page with no groups), I tracked it down, saved
the file probably twice to test some code, it worked without error, I
posted it and promptly updated to the latest devel version which overwrote
my changes lol... At least it wasn't a complete waste of time this way ;)

File: src/usr/local/www/status_gateway_groups.php [hunk 1]
```diff
@@ -35,11 +35,12 @@
 
 require_once("guiconfig.inc");
 
-if (!is_array($config['gateways']['gateway_group'])) {
+if (!is_array($config['gateways']['gateway_group'] || !is_array($config)['gateways'])) {
```
[the remark this answers is not in the document]
Comment (gisforgirard) [reply src/usr/local/www/status_gateway_groups.php]: It was trying to reference a string as if it were an array and giving a blank white fatal error page

**Comment by gisforgirard** [comment]
$config['gateways'] for some reason was a string - it was trying to reference ['gateway_group'] which throws a fatal error - there's no way to reference a string like that, so I tossed in a test that $config['gateways'] is an array, magic, all good!

**Pull request closed without merging**